docs/about-mono/releases/5.2.0.md

```diff
@@ -27,6 +27,16 @@ To debug the assembly loading process, set the logging environment variables to
 
 The current default behavior is the same as in previous Mono versions. It is explicitly available with the `--assembly-loader=legacy` option.
 
+Mangler V1
+---------------------
+
+An initial implementation of the name mangler was created. It associates with each method a unique string that captures the unique attributes of the method. If two methods have the same mangled representation, they're the same method.
+
+Safer Getenv 
+---------------------
+
+The g_getenv and g_setenv are neither threadsafe nor reentrant. Subsequent calls to g_getenv can cause previous strings returned from it to point to garbage memory. This was leading to issues with System.Configuration that could be seen with HttpClient. Now we lock around these calls and return duplicated memory. This is a serious API change to g_getenv, as the returned memory must now be freed.
```

Comment:
with `serious API change` do you mean for embedders? this should have more details about what to change then.